libcloud/security.py

```diff
@@ -61,6 +65,19 @@
     '/etc/ssl/certs/YaST-CA.pem',
 ]
 
+# Insert certifi CA bundle path to the front of Libcloud CA bundle search
+# path if certifi is available
+try:
+    import certifi
+except ImportError:
+    has_certifi = False
+else:
+    has_certifi = True
+
+if has_certifi and USE_CERTIFI:
+    certifi_ca_bundle_path = certifi.where()
+    CA_CERTS_PATH.insert(0, certifi_ca_bundle_path)
```

Comment:
The thing is that `USE_CERTIFI` actually defaults to True right now. So in this case issuing a warning would be confusing.

I'm personally not totally opinionated on this one (yet) and I could be convinced to default it to False and issue a warning if set to True.